packages/v4/patternfly-docs/content/accessibility/menu-toggle/menu-toggle.md

```diff
@@ -0,0 +1,56 @@
+---
+id: Menu toggle
+section: components
+---
+
+import { Checkbox, List, ListItem } from '@patternfly/react-core';
+
+
+## Accessibility
+
+To implement an accessible PatternFly **menu toggle** component:
+- Due to the composable nature of the menu and menu toggle components, keyboard interaction will need to be added manually between the menu toggle and the menu. See the [composable menu demos](/demos/composable-menu) to see how this is done.
+- When adding an action to the menu toggle, an `aria-label` will need to be added to the `MenuToggleAction` or the `MenuToggleCheckbox` to communicate to users what this action will do.
+- When adding any additional elements to the menu toggle, remember to follow that element's specific considerations. For example, if you are adding an avatar to the menu toggle, you will need to provide an `alt` attribute with a description of the image.
+
+
+## Testing
+
+At a minimum, a menu toggle component should meet the following criteria:
+
+<List isPlain>
+  <ListItem>
+    <Checkbox id="menutoggle-a11y-checkbox-1" label={<span>Users should be able to use keyboard navigation to activate the menu toggle with <kbd>Enter</kbd> or <kbd>Space</kbd> to enter a menu.</span>} description={<span><kbd>Tab</kbd> navigates to the menu and <kbd>Shift</kbd> + <kbd>Tab</kbd> navigates to the previous focusable element. Refer to our <a href="/components/menu/accessibility">menu accessibility documentation</a> for more implementation details when using these components together. </span>}/>
+  </ListItem>
+  <ListItem>
+    <Checkbox id="menutoggle-a11y-checkbox-2" label={<span>If there is not text within the menu toggle, such as a kebab menu toggle, an <code className="ws-code">aria-label</code> is used on the <code className="ws-code">MenuToggle</code>.</span>} description="This will communicate what the menu toggle is to users of assistive technologies when navigating through a page or a screen reader's rotor menu." />
+  </ListItem>
+  <ListItem>
+    <Checkbox id="menutoggle-a11y-checkbox-3" label={<span>Any action used within a menu toggle has an <code className="ws-code">aria-label</code>.</span>} description="This will differentiate each action when announced to users of assistive technologies when navigating through a page or a screen reader's rotor menu." />
+  </ListItem>
+</List>
+
+
+## React customization
+
+The following React props have been provided for more fine-tuned control over accessibility.
+
+| Prop | Applied to | Reason | 
+|---|---|---|
+| `aria-label="[text describing the menu toggle]"` | `MenuToggle` | Adds an accessible name to the menu toggle. |
+| `icon={[custom icon]}` | `MenuToggle` | Adds an optional icon rendered inside the toggle, before the children content. Add `aria-hidden` to the icon to hide the decorative item from screen reader elements. |
+| `aria-label="[text describing the action]"` | `MenuToggleAction` | Adds an accessible name to the menu toggle action. |
```

Comment:
Can we update this to say something like "Adds an accessible name to the menu toggle action when the action is represented by an icon with no text"